Documentation/kube-aws-cluster-updates.md

```diff
@@ -0,0 +1,40 @@
+# kube-aws cluster updates
+
+## Types of cluster update
+There are two distinct categories of cluster update.
+
+* **Parameter-level update**: Only changes to `cluster.yaml` and/or TLS assets in `credentials/` folder are reflected. To enact this type of update. Modifications to CloudFormation or cloud-config userdata templates will not be reflected. In this case, you do not have to re-render:
+
+```sh
+kube-aws up --update
+```
+
+* **Full update**: Any change (besides changes made to the etcd cluster- more on that later) will be enacted, including structural changes to CloudFormation and cloudinit templates. This is the type of upgrade that must be run on installing a new version of kube-aws, or more generally when cloudinit or CloudFormation templates are modified:
+
+```sh
+kube-aws render
+git diff # view changes to rendered assets
+kube-aws up --update
+```
+
+## Certificate rotation
+
+The parameter-level update mechanism can be used to rotate in new TLS credentials:
+
+```sh
+kube-aws render --generate-credentials
+kube-aws up --update
```

Comment:
The docs above made it sound like a parameter-level update is distinguished by not requiring a render, but we're rendering here?

Maybe generate-credentials should be its own command?  Maybe there's no need to distinguish between the methods but just note the special cases where it is required?  I believe re-rendering is always safe, if so, maybe we should just always recommend rendering?

I just feel like this notion of two methods of updating with the same command is confusing.